docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the available configurations for ScalarDB.
 
-## Transaction manager configurations
+## ScalarDB client configurations
 
-ScalarDB has several transaction manager implementations, such as Consensus Commit, gRPC, and JDBC, and you can specify one of the implementations with the `scalar.db.transaction_manager` property.
-This section describes the configurations for each transaction manager.
+ScalarDB provides its own transaction protocol called Consensus Commit. You can use the Consensus Commit protocol directly through the ScalarDB client library.
```

Comment:
This should also be revised since it only describes the direct access to ScalarDB.
Also, I think it's more user-friendly to add a note here to explain that ScalarDB Cluster is a component we provide only in the Enterprise edition.

Reply:
Ah, good catch. How does the following look?:

```suggestion
ScalarDB provides its own transaction protocol called Consensus Commit. You can use the Consensus Commit protocol directly through the ScalarDB client library or through ScalarDB Cluster, which is a component that is available only in the ScalarDB Enterprise edition.
```